app/models/github_issue.rb

```diff
@@ -1,65 +1,78 @@
+# NOTE: we are using `GithubIssue` to store issues, pull requests and comments
 class GithubIssue < ApplicationRecord
+  CATEGORIES = %w[issue issue_comment].freeze
+  API_URL_REGEXP = /\Ahttps:\/\/api.github.com\/repos\/.*\z/.freeze
```

Comment:
This and most other constants here would benefit from the `%r{}` regex literals:

```suggestion
  API_URL_REGEXP = %r{\Ahttps://api.github.com/repos/.*\z}.freeze
```

Reply:
+1 I'll do it with the other ones